Sources/AsyncKit/Exports.swift

```diff
@@ -1,6 +1,4 @@
-@_documentation(visibility: internal) @_exported import class NIOEmbedded.EmbeddedEventLoop
```

Comment:
Just a heads up, these exports are being removed because NIOEmbedded and NIOPosix won't be used anymore in the AsyncKit target. That could affect downstream users. But it doesn't make sense to keep these here if they aren't used anymore.

Not sure if that warrants a minor or major bump in the semver. If so, I can conditionalize this exports using `canImports` to lower the impact risk.